packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,51 @@
+# 50.7.0 - December 02, 2020
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Org browser retrieve perf enhancements ([PR #2756](https://github.com/forcedotcom/salesforcedx-vscode/pull/2756))
+
+## Fixed
+
+#### docs
+
+- Add license note for interactive debugger ([PR #2760](https://github.com/forcedotcom/salesforcedx-vscode/pull/2760))
+
+- Image for opening SOQL Builder ([PR #2753](https://github.com/forcedotcom/salesforcedx-vscode/pull/2753))
+
+- Update beta disclaimer for soql builder docs ([PR #2749](https://github.com/forcedotcom/salesforcedx-vscode/pull/2749))
+
+- Add SOQL Builder to doc site ([PR #2745](https://github.com/forcedotcom/salesforcedx-vscode/pull/2745))
+
+- Auto-complete support for js-meta.xml ([PR #2726](https://github.com/forcedotcom/salesforcedx-vscode/pull/2726))
+
+#### salesforcedx-utils-vscode
+
+- Remove core extension as dependency on SOQL extension part 1 ([PR #2743](https://github.com/forcedotcom/salesforcedx-vscode/pull/2743))
+
+#### salesforcedx-vscode-core
+
+- Remove unused code from generated extensions (vsix) ([PR #2769](https://github.com/forcedotcom/salesforcedx-vscode/pull/2769))
+
+#### salesforcedx-vscode-lightning
+
+- Update aura & lwc language servers to v3.0.12 ([PR #2775](https://github.com/forcedotcom/salesforcedx-vscode/pull/2775))
+
+#### salesforcedx-vscode-lwc
+
+- Replace use of info message on js-meta lwc code ([PR #2777](https://github.com/forcedotcom/salesforcedx-vscode/pull/2777))
+
+- Update aura & lwc language servers to v3.0.12 ([PR #2775](https://github.com/forcedotcom/salesforcedx-vscode/pull/2775))
+
+- Auto-complete support for js-meta.xml ([PR #2726](https://github.com/forcedotcom/salesforcedx-vscode/pull/2726))
+
+#### salesforcedx-vscode-soql
```

Comment:
We can remove everything under this since we won't be publishing the soql extension yet as part of this process.